eng/common/scripts/Create-APIReview.ps1

```diff
@@ -140,7 +140,9 @@ if ($packages)
             else
             {
                 # Return error code if status code is 201 for new data plane package
-                if ($pkgInfo.SdkType -eq "client" -and $pkgInfo.IsNewSdk)
+                # Temporarily enable API review for spring SDK types. Ideally this should be done be using 'IsReviewRequired' method in language side
```

Comment:
Do we expect this to be temporary? If so can you link to a issue adding that new property? Do we think just using IsNewSdk is enough? That will start bringing in mgmt libraries but that might also be desired.

Reply:
Python and few other languages didn't want the overhead of approving management packages so checking just the new SDK flag will be an issue immediately. I will file an issue for long term fix and link here.

Reply:
Filed https://github.com/Azure/azure-sdk-tools/issues/1914